PsfShimMonitor/KernelTrace.cs

```diff
@@ -0,0 +1,1356 @@
+#if DEBUG
+                                                 if (_ModelEventItems.Count < 10000)
+#else
+                                                 if (FilterOnProcessId == pid)
+#endif
+                                                 {
+                                                     try
+                                                     {
+                                                         string inputs = "DiskNumber=    \t0x" + ((int)data.PayloadByName("DiskNumber")).ToString("x");
+                                                         inputs += "\nIrpFlags=    \t" + Interpret_KernelFile_IrpFlags((Microsoft.Diagnostics.Tracing.Parsers.Kernel.IrpFlags)data.PayloadByName("IrpFlags"));
+                                                         inputs += "\nPriority=    \t" + Interpret_KernelFile_Priority((Microsoft.Diagnostics.Tracing.Parsers.Kernel.IOPriority)data.PayloadByName("Priority"));
+                                                         inputs += "\nTransferSize= \t0x" + ((int)data.PayloadByName("Priority")).ToString("x");
+                                                         inputs += "\nByteOffset=   \t0x" + ((long)data.PayloadByName("ByteOffset")).ToString("x");
+                                                         inputs += "\nIrp=      \t0x" + ((ulong)data.PayloadByName("Irp")).ToString("x");
+                                                         inputs += "\nFileKey=    \t0x" + ((ulong)data.PayloadByName("FileKey")).ToString("x");
+
+                                                         string outputs = "ElapsedTimeMS=\t" + data.PayloadStringByName("ElapsedTimeMS");
+                                                         outputs += "\nDiskServiceTimeMS=\t" + data.PayloadStringByName("DiskServiceTimeMSec");
+
+
+                                                         EventItem ei = new EventItem(data, inputs, "", outputs, "");
+                                                         lock (_TKernelEventListsLock)
+                                                         {
+                                                             _TKernelEventListItems.Add(ei);
+                                                         }
+                                                         worker.ReportProgress((int)data.EventIndex);
+                                                     }
+                                                     catch (Exception ex)
+                                                     {
+                                                         ;
+                                                     }
+                                                 }
+                                             }
+                                             else if (data.EventName.StartsWith("DiskIO/FlushInit"))
+                                             {
+  
```

Comment:
Is this method being used anywhere?  If this method is not being used, please remove it.

Reply:
Unclear what the ask is.  The comment points to a method that clearly is in use.

Reply:
If I search the whole solution for the text"Interpret_return_rom_win32" the only hit is this line right here.  I also did a text search on all files in this directory for "Interpret_return_rom_win32" and that search also showed that the only time "Interpret_return_rom_win32" shows up is on this line.

Based on my results, I don't see any code calling into this method.

Reply:
Oh. The comment is pointing to code elsewhere so I didn't know what method you were addressing.
Yes, this is an obsolete function that should have been removed.  
This has now been removed.